### ISSUE TITLE
Multiple capital letters in title are lowercased and empty space is added in between

### ISSUE THREAD

**Issue by hixus** [ChronicStone/typed-xlsx#4]
Seems multiple capital letters in title are lowercased and empty space is added in between. Tested the sample and noticed the `ID` has space between. Would assume title remains as written by default.

![CleanShot 2024-04-28 at 20 03 07@2x](https://github.com/ChronicStone/typed-xlsx/assets/1814734/b05fbe04-9c14-4384-aa16-26b8bc70217c)

```
interface User {
    id: number;
    name: string;
    email: string;
  }

  const schema = ExcelSchemaBuilder.create<User>()
    .column("ID", { key: "id" })
    .column("NAME", { key: "name" })
    .column("Email", { key: "email" })
    .build();

  const users: User[] = [
    {
      id: 1,
      name: "JOHN Doe",
      email: "john@foo.com",
    },
  ];

  const excelFile = ExcelBuilder.create()
    .sheet("Sheet1")
    .addTable({
      data: users,
      schema,
    })
    .build({ output: "buffer", bordered: false });
``

**Comment by ChronicStone** [comment]
@hixus Thanks for the report, fixed & released with v0.2.12.
When label isn't provided, the lib use the column key as label but previously ran some formatting on the raw key.

It was on the early iterations of the lib, does not seem useful & relevant anymore, I just removed it.

**Comment by hixus** [comment]
Thanks, works now.
